cpu/stellaris_common/license.html

```diff
@@ -0,0 +1,364 @@
+<html>
```

Comment:
Yes, the license if for binary software and not for the header files included. 
I will remove these license files. Its not required. 
It is left from when I stripped stellarisware library to include only these header files and remove all copyright binaries.